grid2op/Opponent/OpponentSpace.py

```diff
@@ -79,7 +79,7 @@ def has_failed(self):
         """
         self.previous_fails = True
 
-    def attack(self, observation, agent_action, env_action):
+    def attack(self, observation, env, agent_action, env_action):
```

Comment:
maybe the env here, but i would avoid, really :-(

Reply:
Do you still need the `env` here ?

Reply:
No :)